src/DbTests/SQL/ManyColumns.sql

```diff
@@ -0,0 +1,601 @@
+﻿SELECT 
```

Comment:
Could you replace this script with the following, which does the same in a lot fewer lines?

```sql
DECLARE @numCols INT = 600
DECLARE @sql NVARCHAR(MAX) = 'SELECT '

DECLARE @colNo INT = 1

WHILE (@colNo <= @numCols)
BEGIN
  SET @sql += 'Column' + CAST(@colNo AS NVARCHAR) + ' = NULL' + CASE @colNo WHEN @numCols THEN '' ELSE ',' END
  SET @colNo += 1
END

EXEC sp_executesql @sql
```

Reply:
Code swapped out